src/app/screens/Settings.tsx

```diff
@@ -77,6 +78,22 @@ function Settings() {
           )}
         </Setting>
 
+        <Setting
+          title="Legacy LNURL-Auth"
+          subtitle="The key generation for LNURL-auth has changed. Due to an error Alby was not compatible with other implementations. This was fixed, but now different keys are used. If you used LNURL-auth to login before you can still enable the old method. This option will be removed later, make sure to switch to the new login."
+        >
+          {!loading && (
+            <Toggle
+              checked={settings.legacyLnurlAuthKey}
+              onChange={() => {
+                saveSetting({
+                  legacyLnurlAuthKey: !settings.legacyLnurlAuthKey,
```

Comment:
This is just a boolean , right? `legacyLnurlAuthKey` could sound like it's a key. Maybe `isUsingLegacyLnurlAuthKey` could make this more clear?

Reply:
the other one `legacyLnurlAuth` is also wrong, because that actually has nothing to do with lnurl auth :) 

yep, isUsingLegacyLnurlAuthKey sounds good. 

though I hope that we soon remove this